wp-content/themes/csisnuclearnetwork/assets/_scss/pages/post.scss

```diff
@@ -133,10 +133,58 @@
 
 .post {
   &__authors {
-    margin-top: rem(48);
+    margin-top: rem(24);
     margin-bottom: rem(32);
-    padding-top: rem(24);
-    border-top: 1px solid $color-black-130;
+    padding-top: rem(56);
+
+    &::before {
+      content: '';
+      position: relative;
+      width: 100%;
+      padding: 56px 100%;
+      border-top: 1px solid $color-accent-primary-blue-100;
+    }
+
+    &-heading {
+      margin-bottom: rem(16);
+      color: $color-accent-primary-blue-100;
+      @extend %text-style-label-large-strong-caps;
+    }
+
+    &-disclaimer {
```

Comment:
This has line height and letter spacing as well.

Reply:
resolved